tests/servers/test_liveserverthread.py

```diff
@@ -0,0 +1,28 @@
+from django.db import DEFAULT_DB_ALIAS, connections
+from django.test import LiveServerTestCase, TestCase
+
+
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+
+    def test_closes_connections(self):
+        conn = connections[DEFAULT_DB_ALIAS]
```

Comment:
No, `connections[DEFAULT_DB_ALIAS]` has type `django.db.backends.sqlite3.base.DatabaseWrapper`, and `connection` has type `django.db.DefaultConnectionProxy`. Using the latter resulted in the following error:

      File ".../django/test/testcases.py", line 1270, in run
        connections.close_all()
      File ".../django/db/utils.py", line 234, in close_all
        connection.close()
      File ".../django/db/__init__.py", line 36, in __getattr__
        return getattr(connections[DEFAULT_DB_ALIAS], item)
      ...
      File ".../django/db/__init__.py", line 36, in __getattr__
        return getattr(connections[DEFAULT_DB_ALIAS], item)
    RecursionError: maximum recursion depth exceeded while calling a Python object

One way to look at it is this--

You can view `connections_override` as replacement values for `django.db.connections`, so it makes sense that `connections_override[DEFAULT_DB_ALIAS]` be set to `connections[DEFAULT_DB_ALIAS]` (to have the right type) as opposed to `django.db.connection`.